public/index.html

```diff
@@ -2,15 +2,58 @@
 <html>
     <head>
         <meta charset="utf-8">
-        <meta name="description" content="Audacious Arets">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <title>Untitled</title>
-        <link rel="author" href="humans.txt">
+        <title>Facesmash</title>
+        <link rel="stylesheet" href="styles.css">
     </head>
     <body>
-        <h1>Audacious Aretes</h1>
-        <ul>
-            <li>Mychelle Blake</li>
-        </ul>
+ 
+            <ul class="navlist">
+                <li><h2>facesmash</h2>
+                    <ul>
+                        <li><a href="#">about</a></li>
+                        <li><a href="#">sign up</a></li>
+                        <li><a href="#">login</a></li>
+                    </ul>
+                </li>
+            </ul>
+   
+            <div class="loginform">
+                <form method="post" action="mailto:mychelle.blake@gmail.com" name="Loginform">
+                    <p>Email or Phone&nbsp;&nbsp;<input type="text" name="LoginID"/></p>
+                    <p>Password&nbsp;&nbsp;<input type="password" name="Password"/></p>
+                    <p>Log In&nbsp;&nbsp;<input type="submit" value="Log In"/></p>
+                </form>
+            </div>
+ 
+        <div id="container">
+
+        <div id="main">
+            <p>this is the main area</p>
+        </div>
+
+        <div id="sidebar">
+            <h1>Sign Up</h1>
+            <p>It's free and always will be.</p>
+             <form method="post" action="mailto:mychelle.blake@gmail.com">
+                <p>First Name&nbsp;&nbsp;<input type="text" name="firstname"/>Last Name&nbsp;&nbsp;<input type="text" name="lastname"/></p>
```

Comment:
Since these aren't really paragraphs, `div` tags are best used here to wrap label + input. also, be sure to look at the label element I mentioned in Slack.